Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -306,15 +333,15 @@ private void InitializePosition()
                         break;
                     case SearchWindowAligns.CenterTop:
                         Left = HorizonCenter(screen);
-                        Top = 10;
+                        Top = VerticalTop(screen);
```

Comment:
Because this value is a fixed value, there is a problem with vertical alignment monitor situation. Try to locate the window and set the position.